packages/neo-one-client/src/user/LocalUserAccountProvider.ts

```diff
@@ -823,6 +841,172 @@ export class LocalUserAccountProvider<TKeyStore extends KeyStore, TProvider exte
     );
   }
 
+  private async consolidate({
+    inputs,
+    transactionUnsignedIn,
+    from,
+    monitor,
+    byteLimit,
+  }: {
+    readonly transactionUnsignedIn: TransactionModel;
+    readonly inputs: ReadonlyArray<UnspentOutput>;
+    readonly from: UserAccountID;
+    readonly monitor?: Monitor;
+    readonly byteLimit: number;
+  }): Promise<TransactionModel> {
+    const newInputSize = 68;
+    const newOutputSize = 120;
+    const messageSize = transactionUnsignedIn.serializeUnsigned().toString('hex').length;
+
+    if (byteLimit - messageSize < 0) {
+      throw new TransactionSizeExceedsLimitError(messageSize, byteLimit);
+    }
+    if (byteLimit - messageSize - newInputSize < 0) {
+      return transactionUnsignedIn;
+    }
+
+    const inputMap = (): { [key: string]: ReadonlyArray<UnspentOutput> } => ({});
+    const inputArray = (): ReadonlyArray<InputModel> => [];
+    const outputArray = (): ReadonlyArray<OutputModel> => [];
+    const getMessageSize = (numInputs: number, numOutputs: number) =>
+      messageSize + numInputs * newInputSize + numOutputs * newOutputSize;
```

Comment:
Change this to `messageSize + (numInputs * newInputSize) + (numOutputs * newOutputSize)` to make the order of operations clearer.

Reply:
I actually can't, when I save it formats it like this. Originally I had tried to do it with parentheses.